modules/programs/taskwarrior.nix

```
@@ -99,5 +101,21 @@ in {
 
       ${cfg.extraConfig}
     '';
+
+    home.activation.regenDotTaskRc = hm.dag.entryAfter [ "writeBoundary" ] ''
+      $VERBOSE_ECHO "Ensuring generated taskwarrior config included in taskrc"
+
+      if [[ ! -s "${userConf}" ]]; then
+        # Ensure file's existence
+        if [[ -v DRY_RUN ]]; then
+          $DRY_RUN_CMD echo "include ${homeConf}" ">" "${userConf}"
+        else
+          echo "include ${homeConf}" > "${userConf}"
+        fi
+      elif ! ${pkgs.gnugrep}/bin/grep -qF "include ${homeConf}" ${userConf}; then
+        # Add include statement for home-manager generated config
+        $DRY_RUN_CMD ${pkgs.gnused}/bin/sed -i '1i include ${homeConf}' ${userConf}
```

```suggestion
      elif ! ${pkgs.gnugrep}/bin/grep -qF "include ${homeConf}" ${escapeShellArg userConf}; then
        # Add include statement for home-manager generated config
        $DRY_RUN_CMD ${pkgs.gnused}/bin/sed -i '1i include ${homeConf}' ${escapeShellArg userConf}
```

Missed this.
Do we need to escape the `homeConf` path in the user configuration?

---

Worked on my machine without... Can test on a 2nd machine if you like - but I think somone else needs to weigh in here really.

---

Got around to test on a different machine, works as-is (read: without your above suggested edit) on two separate machines for me.

No escaping necessary. I vote for not merging until we find/trigger a need?